modules/@ergonode/attributes/src/locales/en_GB.json

```diff
@@ -45,6 +56,21 @@
             "deleteRequest": "Attribute hasn’t been deleted",
             "optionUniq": "Option code must be unique",
             "optionEmpty": "Option cannot be empty"
+        },
```

Comment:
We've gotta think how to add translations from extended components which comes from outside modules.

Reply:
it's already solved, VueMS mechanism connects all translation files